sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/TwinAsyncTests.java

```diff
@@ -474,4 +474,59 @@ public void deleteTwinSucceedsWhenETagMatches(HttpClient httpClient, DigitalTwin
             }
         }
     }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.digitaltwins.core.TestHelper#getTestParameters")
+    @Override
+    public void digitalTwinWithNumericStringProperty(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion) throws JsonProcessingException {
```

Comment:
It looks like we have 2 positive test cases added, but no negative test cases.

Reply:
I will write unit tests for the serializer and capture the negative test cases. how does that sound?